src/replica/replica_stub.h

```diff
@@ -135,7 +135,7 @@ class replica_stub : public serverlet<replica_stub>, public ref_counter
     void on_add_learner(const group_check_request &request);
     void on_remove(const replica_configuration &request);
     void on_group_check(group_check_rpc rpc);
-    void on_copy_checkpoint(copy_checkpoint_rpc rpc);
+    void on_query_last_checkpoint(query_last_checkpoint_info_rpc rpc);
```

Comment:
Please add some comments to should it is used for duplication, same as the function definition in `replica.h`.